### PR TITLE
feature/setup-github-actions-to-the-project

### PR DESCRIPTION
**Pull request by topsart**
Setup GitHub Actions to the project (https://trello.com/c/kDUWkmZZ/13-setup-github-actions-for-the-project)

- created .github/workflows/ directory with actions.yml file in it
- configured .yml file
- added job with tests and deployment for the project

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: deployment
+
+on:
+  push
+  # if u want to push only on master branch
+  # push:
+  #   branches:
+  #     - master
```
Comment (kerrimov) [inline .github/workflows/actions.yml:8]: Why do you leave this comment here?

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+on:
+  push
+  # if u want to push only on master branch
+  # push:
+  #   branches:
+  #     - master
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
```
Comment (kerrimov) [inline .github/workflows/actions.yml:12]: Linux should as default OS, isn’t it?

**Comment by kerrimov** [comment]
Why did you create .github/workflows/ folder but not put the file in the src folder?

File: .github/workflows/actions.yml (hunk 2 above)
Comment (topsart) [reply .github/workflows/actions.yml:12]: Note from docs

image

Comment (kerrimov) [reply .github/workflows/actions.yml:12]: yep, please test

File: .github/workflows/actions.yml (hunk 1 above)
Comment (topsart) [reply .github/workflows/actions.yml:8]: removed

**Review by kerrimov — APPROVED** [review]

**Pull request merged**